setup.py

```diff
@@ -13,10 +14,11 @@
         'requests',
```

Comment:
This needs changing to `requests[security]` as this is what is used when we do `pip install salt-shaker` – it won't look at requirements.txt in that case.

We should probably remove requirements.txt entirely.